### PR TITLE
clarify basic example in README

### PR DESCRIPTION
**Pull request by duranje**
## Description
The 'Getting Started' section includes a breakdown of each component and its role. The 'Code Sample' is now a working example. 

## Motivation and Context
We use this library at work in a number of projects and have made a few updates to the documentation that I thought others might find useful. If the maintainers don't deem it so, no worries!

## How Has This Been Tested?
-Asked other devs if they found the information helpful
-copied and pasted the code sample into a freshly-bootstrapped create-react-app project to verify it works

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,12 +36,94 @@ npm install fixed-data-table-2
...
+```
+The cell components in a column will receive the current array index of your data as a prop (`this.props.rowIndex`). Use this to access the correct value for each cell.
+```javascript
+    const rows = [0,1,2];
+    
+    <Column
+      header={<Cell>Column 1</Cell>}
+      cell={({rowIndex, ...props}) => (
+        <Cell {...props}>
+        {rows[rowIndex] }
```
Comment (wcjordan) [inline README.md]: nit space before rows

File: README.md [hunk 2]
```diff
@@ -73,7 +161,7 @@ ReactDOM.render(
       header={<Cell>Col 3</Cell>}
       cell={({rowIndex, ...props}) => (
         <Cell {...props}>
-          Data for column 3: {rows[rowIndex][2]}
+          Data for column 3: {rows[rowIndex]}
```
Comment (wcjordan) [inline README.md]: If we do this, could you change rows to just be an array of values instead of array of arrays

**Comment by wcjordan** [comment]
Merged, I'll update the site docs shortly to reflect the changes

**Comment by duranje** [comment]
Thanks for maintaining this library-we’ve found it very useful!

**Pull request merged**